package.json

```
@@ -25,7 +25,11 @@
     "intFDTest": "mocha --grep INT-FD/ test/intTest.js",
     "intFDRTest": "mocha --grep INT-FDR/ test/intTest.js",
     "intSIGVALITest": "mocha --grep INT-SIGVALI/ test/intTest.js",
-    "intAPITest": "mocha --grep INT-API/ test/intTest.js"
+    "intAPITest": "mocha --grep INT-API/ test/intTest.js",
+    "cover": "npm run build && nyc --silent mocha test/packages/caver.utils.js && nyc --silent --no-clean mocha test/packages/caver.klay.net.js && npm run cover:scenario && npm run cover:wallet && npm run cover:report",
+    "cover:scenario": "nyc --silent --no-clean npm run serTest",
```

```suggestion
    "cover:serialization": "nyc --silent --no-clean npm run serTest",
```
npm run serTest tests serialization :) So i think cover:serialization is better.